settings.py

```diff
@@ -316,6 +316,8 @@
 MME_SLACK_MATCH_NOTIFICATION_CHANNEL = 'matchmaker_matches'
 MME_SLACK_SEQR_MATCH_NOTIFICATION_CHANNEL = 'matchmaker_seqr_match'
 
+SEQR_SLACK_DATA_ALERTS_NOTIFICATION_CHANNEL = 'seqr_loading_pipeline'
```

Comment:
not sure this is a good channel for these alerts, since I think we are trying to move responsibility for managing IGV issues to the PM team. Can you post in the channel asking people there if they think this would be a good channel for them/ which channel they would recommend?